js/entryView.js

```diff
@@ -1,5 +1,7 @@
 var entryView = {};
 
+// This code is here so for when and if I add the select menus
+
```

Comment:
It's good that you have a comment here explaining this commented-out code. Best practice for maintaining clean code is to leave provisional and experimental features in other version control branches such as `development`, or in this case the `class-03` branch.